### PR TITLE
Update incremental import note as it supports the block format

### PR DESCRIPTION
**Pull request by NataliaIvakina**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1599.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1599.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

File: modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc [hunk 1]
```diff
@@ -465,7 +465,7 @@ If importing to a database that has not explicitly been created prior to the imp
 
 [WARNING]
```
Comment (renetapopova) [inline modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc:466]:
```suggestion
[NOTE]
```

**Review by renetapopova — APPROVED** [review]
Looks good — just a tiny suggestion.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1599.surge.sh

Updated pages:
- [Import](https://neo4j-docs-operations-1599.surge.sh/operations-manual/5/tools/neo4j-admin/neo4j-admin-import)

**Pull request merged**